Task_Array/index.js

```diff
@@ -1,67 +1,68 @@
-let arr = [1, 3, 2, 5, 4];
-
-// Filter the odd number using the filter function of the array and store it in filterArr declared below.
-
-let filterArr;
```

Comment:
The variable 'filterArr' is declared twice. Remove line 5 to avoid redeclaration.